contracts/cosmos/verifier/lib/nim/contractInteraction.nim

```diff
@@ -0,0 +1,58 @@
+import
+  std/[os,osproc,strutils],
+  helpers,
+  confutils,
+  config
+
+proc init*(pathVerificationKey, pathCurrentHeader, code_id, wallet, node, txflags: string): string =
+  let vkey = createVerificationKey(pathVerificationKey)
+  let currentHeader = createCurrentHeader(pathCurrentHeader)
```

Comment:
Information about the current header should be downloaded right before the deployment